index.js

```diff
@@ -1,6 +1,6 @@
 require('dotenv').config();
 const fetch = require('node-fetch');
-const Moment = require('moment');
+const Moment = require('moment-timezone');
```

Comment:
Should there be a package.json change that goes with this?

Reply:
done